javascript/ql/src/semmle/javascript/security/dataflow/DomBasedXss.qll

```diff
@@ -8,15 +8,23 @@ import javascript
 module DomBasedXss {
   import DomBasedXssCustomizations::DomBasedXss
 
+  /**
+   * DEPRECATED. Use `HtmlInjectionConfiguration` or `JQuerySelectorInjectionConfiguration`.
+   */
+  deprecated class Configuration = HtmlInjectionConfiguration;
```

Comment:
Have you considered mentioning this deprecation explicitly in the change notes? I would expect this particular `Configuration` instance to be the one that is most commonly imported in external queries.